assets/js/blocks/cart/inner-blocks/cart-cross-sells-products/block.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { useStoreCart } from '@woocommerce/base-context/hooks';
+
+/**
+ * Internal dependencies
+ */
+import CartCrossSellsProductList from '../../cart-cross-sells-product-list';
+import metadata from './block.json';
+
+interface BlockProps {
+	className?: string;
```

Comment:
Because of https://github.com/woocommerce/woocommerce-blocks/blob/trunk/tsconfig.base.json#L10 we need to allow undefined here.

```suggestion
	className?: string | undefined;
```